docs/dev/test-configuration-options.md

```diff
@@ -345,6 +360,20 @@ Sets idle test timeout in seconds. As a safety measure to prevent tests from run
 
 ---
 
+### `devTools`
+
+<p><small>| OPTIONAL | BOOLEAN | <span className="sauceGreen">Desktop Only</span> | <span className="sauceGreen">BETA</span> | </small></p>
+
+Enables [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/) support, which is disabled by default on Sauce Labs platform. This allows Selenium 4 clients to use [Bi-Directional functionality](https://www.selenium.dev/documentation/webdriver/bidirectional/). In particular, this capability exposes the WebSocket endpoint which is available under `se:cdp` field in session startup response body. This endpoint can be used to issue WebDriver BiDi commands as described by the [specification](https://w3c.github.io/webdriver-bidi/). The default value is `false`.
+
+The `devTools` capability is **not compatible** with [`extendedDebugging`](#extendeddebugging) capability.
```

Comment:
does this mean this has to be turned off? or we are ignoring this?

Reply:
@kristofmuhi currently the behavior is not defined, but yes, I think we will need to disable `extendedDebugging` in this case to be extra safe.